src/main/java/com/github/ilim/backend/ilimEmailDeliveryServ/model/EmailRequest.java

```diff
@@ -0,0 +1,73 @@
+package com.github.ilim.backend.ilimEmailDeliveryServ.model;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+/**
+ * Data Transfer Object (DTO) representing an email request.
+ * It encapsulates the necessary details required to send an email.
+ *
+ * <p>This class includes validation annotations to ensure that all
+ * required fields are provided and adhere to the expected formats.</p>
+ *
+ * <p>Author: praveendran</p>
+ */
+public class EmailRequest {
+
+    @NotBlank(message = "To address is mandatory")
+    @Email(message = "To address should be a valid email")
+    private String toAddress;
+
+    @Email(message = "CC address should be a valid email")
+    private String ccAddress; // Optional field
+
+
+    @NotBlank(message = "Subject is mandatory")
+    private String subject;
+
+    @NotBlank(message = "Content is mandatory")
+    private String content;
+
+    // Constructors
+    public EmailRequest() {
+    }
+
+    public EmailRequest(String toAddress, String ccAddress, String fromAddress, String subject, String content) {
```

Comment:
I think you forgot to delete `fromAddress`  parameter after removing it from the class.

Reply:
Oops removed it